identity-account/README.md

```diff
@@ -0,0 +1,40 @@
+# IOTA Identity - Account
```

Comment:
There are still `cargo doc` warnings.

E.g.

```rust
warning: unresolved link to `crypto`
  --> identity-account\src\error.rs:21:34
   |
21 |   /// Caused by errors from the [crypto] crate.
   |                                  ^^^^^^ no item named `crypto` in scope
```

Run `cargo doc --no-deps --open` to see and fix them.

Reply:
I fixed all the account related ones.